deploy/ansible/roles/drand/templates/docker-compose.yml.j2

```diff
@@ -12,9 +12,9 @@ services:
       - "0.0.0.0:{{ drandNumber }}235:8081"
       - "0.0.0.0:{{ drandNumber }}236:1236"
     environment:
-      - DRAND_PUBLIC_ADDRESS=drand-01.pre-devnet.shimmer.iota.cafe:{{ drandNumber }}234
+      - DRAND_PUBLIC_ADDRESS={{ inventory_hostname }}:{{ drandNumber }}234
     entrypoint: /usr/local/bin/entrypoint.sh
-    command: start --verbose --private-listen 0.0.0.0:8000 --public-listen 0.0.0.0:8081 --tls-disable --goshimmerAPIurl "http://drng-0{{ drandNumber }}.pre-devnet.shimmer.iota.cafe:9101" --instanceID 1 --metrics "0.0.0.0:1236"
+    command: start --verbose --private-listen 0.0.0.0:8000 --public-listen 0.0.0.0:8081 --tls-disable --goshimmerAPIurl "http://{{ goshimmerHostTemplate | replace('{drandNumber}', drandNumber) }}:9101" --instanceID 1 --metrics "0.0.0.0:1236"
```

Comment:
Can we change this `goshimmerHostTemplate` to `drngHostTemplate`, as, as I understand, you only use it to iterate over the drngs hosts.

Reply:
Yes. I like that